data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,23 @@
 [
   {
+    "id": "cooking_standard",
+    "type": "requirement",
+    "//": "Cooking of standard meals",
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [
+      [ "fire", -1 ],
+      [ "hotplate", 10 ],
+      [ "mess_kit", 10 ],
+      [ "mil_mess_kit", 10 ],
+      [ "oil_cooker", 10 ],
+      [ "survivor_mess_kit", 10 ],
+      [ "gasoline_cooker", 10 ],
+      [ "charcoal_cooker", 10 ],
+      [ "popcan_stove", 10 ],
+      [ "toolset", 10 ]
+    ] ]
+  },
```

Comment:
When you rewrote some of the burning code is there anywhere I can get even a rough idea of how long something might burn for?

> Actually isn't. A proper fix needs to be more usable than "oops fire went out, tough shit, crafting failed".

It could suspend the activity? That's non-hostile and intuitive. IRL you'd go and put a few more logs on the fire.